### PR TITLE
Add dax to yaml converter

### PR DESCRIPTION
**Pull request by borismattijssen**

### CODE REVIEW

File: dax-to-yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# DAX to YAML converter
```
Comment (nov1n) [inline dax-to-yaml/README.md]: Change all occurrences of YAML to JSON

Comment (nov1n) [inline dax-to-yaml/README.md]: Change folder name to json

Comment (nov1n) [inline dax-to-yaml/README.md]: Add comments

**Comment by borismattijssen** [comment]
Adressed comments. I will split the main file up into smaller pieces to make SIG happy. I will do this later. Can we for now merge this implementation?

**Pull request merged**